llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -664,6 +676,9 @@ class VPBlockBase {
   /// the cloned recipes, including all blocks in the single-entry single-exit
   /// region for VPRegionBlocks.
   virtual VPBlockBase *clone() = 0;
+
+  /// Compute the cost of the block.
+  virtual InstructionCost cost(ElementCount VF, VPCostContext &Ctx) = 0;
```

Comment:
nit: better place all `cost()` methods next to `execute()`, as done in VPRecipeBase.

Reply:
Done, thanks!